src/quo/components/buttons/slide_button/view.cljs

```diff
@@ -71,14 +69,16 @@
                                                 (dimensions :thumb))
                                        [dimensions])
         custom-color                  (if (= type :danger) :danger customization-color)
-        gesture                       (rn/use-memo #(drag-gesture x-pos
-                                                                  disabled?
-                                                                  (dimensions :usable-track)
-                                                                  sliding-complete?
-                                                                  set-sliding-complete
-                                                                  on-complete
-                                                                  reset-fn)
-                                                   [sliding-complete? disabled?])]
+        gesture                       (rn/use-memo
+                                       (fn []
+                                         (drag-gesture x-pos
+                                                       disabled?
+                                                       (dimensions :usable-track)
+                                                       sliding-complete?
+                                                       set-sliding-complete
+                                                       on-complete
+                                                       reset-fn))
+                                       [sliding-complete? disabled? on-complete])]
```

Comment:
This is the main fix for this PR, due to maybe excesive memoization.

CC: @ilmotta

Reply:
@ulisesmac, I would be surprised if excessive memoization was a problem simply because I never heard of this limitation. As long as there's enough memory, I would assume it would work.

Shouldn't we also pass the other args (`x-pos`, `set-sliding-complete`, etc) to the deps vector?

Reply:
@ilmotta oh, I meant that we are trying to memoize everything and cases like this happen (this is the second time I find this kind of issue).

> Shouldn't we also pass the other args (x-pos, set-sliding-complete, etc) to the deps vector?

I already considered it but seems they not needed since they are created internally, `on-complete` is an external value. I also think the memoization would be useless if everything was added to the vector.

Reply:
> oh, I meant that we are trying to memoize everything and cases like this happen (this is the second time I find this kind of issue).

Sounds like an interesting problem @ulisesmac. Let's hope we can figure it out eventually with more examples creeping up. But doesn't sound like a bug React would have, so it's probably our fault at some layer.

> I also think the memoization would be useless if everything was added to the vector.

I think the main benefit for us in caching functions is to reduce the cost of Reagent processing hiccup needlessly while the values are passed down the tree. But here the props are passed directly to the leaf `gesture-detector`. Just something I think sometimes, but never investigated further.